### PR TITLE
Issue #1752 by alexfinnarn: Automatically include a local settings fi…

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/1752#issuecomment-201665135

Replaces https://github.com/backdrop/backdrop/pull/1306

### CODE REVIEW

File: settings.php [hunk 1]
```diff
@@ -34,8 +34,8 @@
  * $config_directories['staging'] = '/home/myusername/config/staging';
  * @endcode
  */
-$config_directories['active'] = 'files/config_' . md5($database) . '/active';
-$config_directories['staging'] = 'files/config_' . md5($database) . '/staging';
+$config_directories['active'] = 'files/config_bed56320d59092fe62f20124c9583997/active';
+$config_directories['staging'] = 'files/config_bed56320d59092fe62f20124c9583997/staging';
```
Comment (quicksketch) [inline settings.php]: LOL. Exactly what we're trying to avoid with this fix. :wink:

Needs to be removed.

File: settings.php [hunk 2]
```diff
@@ -256,7 +255,7 @@
  *
  * Comment out this code if you would like to disable this functionality.
  */
-$settings['404_fast_paths_exclude'] = '/\/(?:styles)|(?:system\/files)\//';
+$settings['404_fast_paths_exclude'] = '/\/(?:styles)\//';
```
Comment (quicksketch) [inline settings.php]: Hm, and an unrelated change here.

File: settings.php [hunk 3]
```diff
@@ -330,3 +329,17 @@
  * built for Backdrop.
  */
 $settings['backdrop_drupal_compatibility'] = TRUE;
+
+
+/**
+ * Include a local settings file.
+ *
+ * To make local development easier, you can add a file that contains settings
+ * for CSS/JS aggregation, caching, or anything else commonly changed during
```
Comment (quicksketch) [inline settings.php]: This documentation is currently incorrect. You can't disable CSS/JS aggregation through settings.php currently. It's config, not a `$settings` option.

Comment (jenlampton) [inline settings.php]: doh. okay, reworked.
